niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -2179,17 +2237,30 @@ public void onError(int error) {
         // android.permission.SMS_FINANCIAL_TRANSACTIONS - both with and without the permission
         // granted a null value can be sent to the callback.
 
+
         mPermissionTasks.put(permission.RESET_PASSWORD,
                 new PermissionTest(false, Build.VERSION_CODES.Q, () -> {
                     String newPassword = null;
                     // In Android 10 a password had to be specified. Since this test only allows a
                     // password to be set if it has not been previously set this password will need
                     // to be cleared before a subsequent test invocation when the test app is
-                    // platform signed.
+                    // platform signed. ag/9703399
```

Comment:
For open source projects, instead of referencing the short link to the change, we need to use the publicly available version; I think something like this should work for this commit: https://cs.android.com/android/_/android/platform/frameworks/base/+/d952240979aea7d10b5f81dfa9199323c79b4363

Reply:
fixed

Reply:
Thanks!